tinker/tinker_controller.py

```diff
@@ -661,18 +660,18 @@ def convert_timestamps_to_bethel_string(self, open, close, all_day):
                 open = proxy_open
                 close = proxy_close
 
-            bethel_date_string = self.final_format(open, close)
+            bethel_date_string = self.final_format(open, close, all_day_format)
             return bethel_date_string
         except:
             return None
 
-    def final_format(self, open, close):
+    def final_format(self, open, close, adf):
```

Comment:
What is the adf variable? It might be helpful to use the full variable name (all_day_format, i assume?)